docs/RFCS/raft_consistency_checker.md

```diff
@@ -0,0 +1,86 @@
+- Feature Name: Raft consistency checker
+- Status: draft
+- Start Date: 2016-02-10
+- Authors: Ben Darnell, Bram Grunier, Vivek Menezes
+- RFC PR: (PR # after acceptance of initial draft)
+- Cockroach Issue: #837
+
+# Summary
+
+An online system that checks that periodically checks that range replicas
+are consistent; a snapshot for a range at a specific point in the raft log
+ has the same sha on all replicas.
```

Comment:
Or s/sha/hash/gi; the specific hash used is an implementation detail (although it should be something from the SHA family; either sha512 from the go standard library or SHA3 from `x/crypto`)